common-api/src/main/java/org/pmiops/workbench/config/WorkbenchLocationConfigService.java

```diff
@@ -23,8 +23,8 @@ public String getWorkbenchAppEngineLocationId() {
    */
   public String getCloudTaskLocationId() {
     String appEngineLocationId = workbenchConfig.get().server.appEngineLocationId;
-    if (appEngineLocationId == "us-central") return "us-central1";
-    else if (appEngineLocationId == "europe-west") return "europe-west1";
+    if (appEngineLocationId.equals("us-central")) return "us-central1";
```

Comment:
Side note: this kind of issue would have been easily caught by a unit test. Be sure to follow-up soon on developing unit tests, which was a condition of approval for https://github.com/all-of-us/workbench/pull/2948. (It might help to track a separate Jira ticket for tests follow-up, so we can include it in sprint pointing). I would recommend including this service as being in scope for unit test coverage.